casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -250,7 +250,7 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
             for {
               _         <- runtime.shortLeashParams.setParams(codeHash, phloPrice, userId, timestamp)
               _         <- runtime.replaySpace.rig(hash, log.toList)
-              injResult <- doInj(deploy, runtime.replayReducer, runtime.errorLog)(runtime.cost)
+              injResult <- doInj(deploy, runtime)
```

Comment:
This line used to forward the `replayReducer` into the `doInj` method. After your change the `doInj` defaults to using the `reducer`

Reply:
good catch! :+1: